R/bias.R

```diff
@@ -113,3 +113,64 @@ bias <- function(true_values, predictions) {
     return(res)
   }
 }
+
+
+
+# lower <- c(6341.000, 6329.500, 6087.014, 5703.500,
+#            5451.000, 5340.500, 4821.996, 4709.000,
+#            4341.500, 4006.250, 1127.000, 705.500)
+# upper <- c(6341.000, 6352.500, 6594.986, 6978.500,
+#            7231.000, 7341.500, 7860.004, 7973.000,
+#            8340.500, 8675.750, 11555.000, 11976.500)
+#
+# range <- c(0, 10, 20, 30, 40, 50, 60, 70, 80, 90, 95, 98)
+# true_value <- 8062
+#
+# quantile_bias(lower = lower, upper = upper,
+#               range = range, true_value = true_value)
+
+
+quantile_bias <- function(range, boundary = NULL,
+                          lower = NULL, upper = NULL,
+                          predictions = NULL, true_value) {
+
+  if (is.null(boundary)) {
```

Comment:
Need to talk through this or see a pointer so some info as I have no idea what is being aimed at here and hence no idea if the code is correct (but it all looks sane) 😄